crates/grafbase-sdk/src/test/runner.rs

```diff
@@ -44,8 +44,9 @@ impl TestRunner {
         let mut subgraphs = Subgraphs::default();
 
         for subgraph in config.mock_subgraphs.drain(..) {
-            subgraphs.ingest_str(subgraph.sdl(), subgraph.name(), subgraph.url().as_str())?;
-            mock_subgraphs.push(subgraph.start());
+            let mock_graph = subgraph.start().await;
```

Comment:
just a nitpick, could be done in parallel

Reply:
But you need to start the subgraph to get its address/port first, and you need that to ingest the subgraph. Kind of not that easy to parallelize, because you mutate the subgraphs...